app/mailers/response_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class ResponseMailer < ActionMailer::Base
+  domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
+  default from: "noreply@#{domain}"
+
+  def response_email(requester_email:, response_uuid:)
+    @link = "future_link_to_finished_response/#{response_uuid}"
+
+    mail to: requester_email,
+         subject: "Resultaten screening #{response_uuid}"
```

Comment:
UUID aan de gebruiker tonen is te technisch, ik zou hier net als bij de inhoud een datum tonen, of zelfs alleen `Resultaten screening`